casbin/rbac/default_role_manager/role_manager.py

```diff
@@ -11,14 +11,29 @@ class RoleManager(RoleManager):
     def __init__(self, max_hierarchy_level):
         self.all_roles = dict()
         self.max_hierarchy_level = max_hierarchy_level
+        self.matching_func = None
+
+    def set_matching_func(self, fn):
```

Comment:
In order to maintain the consistency of each version, it is recommended to use `add_matching_func()`
Refer to: https://casbin.org/docs/en/rbac#use-pattern-matching-in-rbac

Reply:
You are right, it should be consistent. But I would like to propose to rename it to `set_matching_func`(`SetMatchingFunc()`), if it is never going to support more than one matching function.
I will rename it to `add_matching_func()` for now.